### PR TITLE
Node.js Compatibility — update for v2 + unenv polyfills

### PR DESCRIPTION
**Pull request by irvinebroque**
Do not merge yet, draft PR for getting external feedback.

https://bib-unenv.cloudflare-docs-7ou.pages.dev/workers/runtime-apis/nodejs/

### CODE REVIEW

File: content/workers/runtime-apis/nodejs/_index.md [hunk 1]
```diff
@@ -2,61 +2,69 @@
...
+
+When you build your Worker with Wrangler, and enable the `experimental:nodejs_compat_v2` compatibility flag, in addition to enabling built-in Node.js APIs from the Workers Runtime, [Wrangler](/workers/wrangler/) will use [unenv](https://github.com/unjs/unenv) to automatically detect uses of Node.js APIs, and add polyfills where relevant.
+
+Adding polyfills maximizes compatibility with existing npm packages, while recognizing that not all APIs from Node.js make sense in the context of serverless functions.
+
+In cases where it is possible to provide a polyfill version of the relevant Node.js API, unenv will do so. In cases where it is not possible to provide a polyfill, such as the `fs` module, unenv adds the module and its methods to your Worker, but calling methods of the module will either noop, or will throw an error with a message like:
+
+```
+[unenv] <method name> is not implemented yet!
 ```
```
Comment (irvinebroque) [inline content/workers/runtime-apis/nodejs/_index.md]:
```suggestion
```

To understand which Node.js APIs Wrangler can polyfill via unenv, view [this page](https://workers-nodejs-compat-matrix.pages.dev/), which presents a breakdown of which Node.js APIs are supported by the Workers runtime, and which are polyfilled when building Workers with Wrangler.
```

**Comment by mikenomitch** [comment]
Should we have a list of all of the APIs that unenv provides and then mark if they're mocks or have real logic in them? I feel like that'd be helpful.

**Comment by irvinebroque** [comment]
@mikenomitch yes!

File: content/hyperdrive/_partials/_create-hyperdrive-config.md [hunk 2]
```diff
@@ -42,7 +42,8 @@ name = "hyperdrive-example"
 main = "src/index.ts"
 compatibility_date = "2023-09-11"
 
-node_compat = true # required for database drivers to function
+# required for database drivers to function
+compatibility_flags = ["experimental:nodejs_compat_v2"]
```
Comment (jasnell) [inline content/hyperdrive/_partials/_create-hyperdrive-config.md:46]: After 9/2 of this year `nodejs_compat_v2` is equivalent to `nodejs_compat`... should we document that instead?

**Comment by KimJ15** [comment]
👋Hi there, we’re going to close this PR. All PRs prior to August 12 need to be reworked due to the platform migration to Astro.

**Pull request closed without merging**